### PR TITLE
Adding sidebar block to News issue/552

### PR DESCRIPTION
**Pull request by smayden**
This PR adds the Promoted Sidebar block to the az_news view. It also creates a view mode/display for thumbnail to az_image

## Description
This ticket:
- Updates the az_news view with a new block display called "Sidebar News Block" to show stories three most recent stories
- Adds the view mode/display mode "Minimal Media List" that is used by the view
- Adds the view mode/display mode "Thumbnail" that is used by associated media for this view
- Places the block az_news_sidebar to the second sidebar region for pages /news/*

## Related Issue
#552 

## How Has This Been Tested?
Tested locally.

To test changes:
1. Create several news items
2. View a news item and verify that them most recent news items appear in the sidebar on the right
3. Verify that the article you are viewing does not appear in the sidebar stories

image
image

## Types of changes

- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_image.thumbnail.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.media.az_thumbnail
+    - field.field.media.az_image.field_media_az_image
+    - image.style.thumbnail
+    - media.type.az_image
+  module:
+    - image
+id: media.az_image.thumbnail
```
Comment (trackleft) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_image.thumbnail.yml:11]: maybe this line

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_image.thumbnail.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+targetEntityType: media
+bundle: az_image
+mode: thumbnail
```
Comment (trackleft) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_image.thumbnail.yml:14]: and this line

Comment (trackleft) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_image.thumbnail.yml:14]:
```suggestion
mode: az_thumbnail
```

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_image.thumbnail.yml (hunk 1 above)
Comment (trackleft) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_image.thumbnail.yml:11]:
```suggestion
id: media.az_image.az_thumbnail
```

File: themes/custom/az_barrio/config/optional/block.block.views_block__az_news_az_sidebar.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  provider: views
+  label_display: visible
+  views_label: ''
+  items_per_page: none
+  context_mapping: {  }
+visibility:
+  request_path:
+    id: request_path
+    pages: '/news/*'
+    negate: false
```
Comment (trackleft) [inline themes/custom/az_barrio/config/optional/block.block.views_block__az_news_az_sidebar.yaml:30]:
```suggestion
langcode: en
status: true
dependencies:
  config:
    - views.view.az_news
  module:
    - system
    - views
  theme:
    - az_barrio
id: views_block__az_news_az_sidebar
theme: az_barrio
region: sidebar_second
weight: -8
provider: null
plugin: 'views_block:az_news-az_sidebar'
settings:
  id: 'views_block:az_news-az_sidebar'
  label: ''
  provider: views
  label_display: visible
  views_label: ''
  items_per_page: none
  context_mapping: {  }
visibility:
  request_path:
    id: request_path
    pages: '/news/*'
    negate: false
```

**Comment by smayden** [comment]
@joeparsons I reconfigured to move the block file to the themes folder as discussed
@danahertzberg Let me know your thoughts on naming issues you mentioned

Otherwise, I think this is ready for further review!

**Comment by danahertzberg** [comment]
> @danahertzberg Let me know your thoughts on naming issues you mentioned

@smayden I was thinking we could make this appear more reusable if it were named something along the lines of "Small Row view" instead of "Sidebar"...? Just thinking if someone wanted this on their page, it's clear what it is.

Even though Eller uses custom News and Events, I could see this being a common desire where News and Events are displayed next to each other. See the bottom of https://eller.arizona.edu/ as an example.

**Review by barrettbaffert — APPROVED** [review]

**Review by danahertzberg — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**